Lib/shutil.py

```diff
@@ -764,43 +741,16 @@ def onexc(*args):
             path = os.fsdecode(path)
         # Note: To guard against symlink races, we use the standard
         # lstat()/open()/fstat() trick.
```

Comment:
I would move this comment to _rmtree_safe_fd where this code now actually happens.